### DISCUSSION TITLE
Dependent select fields

### DISCUSSION THREAD

**Discussion by SaschaValentin** [pocketbase/pocketbase#2748] [Q&A]
I´m looking for a way to use dependent select fields in Pocketbase in a way like this:

There´s one collection "countries" with the field "name" of different countries and a second collection with fields "cities" and a relation to the country. In a third collection, let´s say "pois", beside fields for the "name" etc., there should also be two relation fields to the conutry and the city, where the poi is placed.

Is it somehow possible to let the second field "city" only show these entries that belong to the country that ist selected before?

**Comment by ganigeorgiev** [comment]
> Is it somehow possible to let the second field "city" only show these entries that belong to the country that ist selected before?

No, this can't be done in the Admin UI at the moment.

Do you really need to have both `country` and `city` relation fields in the `pois` collection? I personally would use only the `city` field since you can retrieve the country from there.

**Reply by SaschaValentin** [reply]
Sure, you´re right. But this was more or less an example to show what I want to achive ;-) A pity this doesn´t work yet. So I have to try to find another way.

Thanks for your help.
